src/curated-corpus/components/ScheduledItemCardWrapper/ScheduledItemCardWrapper.styles.tsx

```diff
@@ -10,6 +10,7 @@ export const useStyles = makeStyles((theme: Theme) =>
       display: 'flex',
       flexDirection: 'column',
       position: 'relative',
+      zIndex: -1,
```

Comment:
sadly this appears to render the buttons on the cards (reschedule, remove) inert 😢